maestro-android/src/androidTest/AndroidManifest.xml

```diff
@@ -4,5 +4,8 @@
 
     <uses-permission android:name="android.permission.ACCESS_MOCK_LOCATION"
                      tools:ignore="MockLocation,ProtectedPermissions"/>
+    <uses-permission android:name="android.permission.QUERY_ALL_PACKAGES"
```

Comment:
from my memory it was only needed for the target app

Reply:
I think so too, but I am putting it in both on the off chance that some Android version might interpret it differently. It doesn't hurt